### PR TITLE
Add synthetics function

### PR DESCRIPTION
**Pull request by tani-yu**
**What this PR does / why we need it**:
I implemented synthetics function.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
implements #39

### CODE REVIEW

File: cmd/synthetics/import.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+)
+
+// syntheticsImportCmd represents the syntheticsImportCmd command
+var syntheticsImportCmd = &cobra.Command{
+	Use:   "import",
+	Short: "Create synthetics on Datadog by importing JSON object",
+	Run: func(cmd *cobra.Command, args []string) {
+		cli, err := dd.NewDDClient()
+		if err != nil {
+			log.Fatalf("failed to connect Datadog API server: %s\n", err)
```
Comment (toVersus) [inline cmd/synthetics/import.go]:
```suggestion
			log.Fatalf("Failed to connect Datadog API server: %s\n", err)
```

File: cmd/synthetics/import.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		}
+
+		var synthetics []datadog.SyntheticsTest
+		for _, inputPath := range args {
+			var decoded []datadog.SyntheticsTest
+			raw, err := ioutil.ReadFile(inputPath)
+			if err != nil {
+				log.Fatalf("failed to read JSON file: %s\n", err)
```
Comment (toVersus) [inline cmd/synthetics/import.go]:
```suggestion
				log.Fatalf("Failed to read JSON file: %s\n", err)
```

File: cmd/synthetics/import.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+			if err := json.Unmarshal(raw, &decoded); err != nil {
+				log.Fatalf("JSON Unmarshal error: %s\n", err)
+			}
+			synthetics = append(synthetics, decoded...)
+		}
+
+		syns, err := cli.GetSyntheticsTests()
+		if err != nil {
+			log.Fatalf("failed to get synthetics items: %s\n", err)
```
Comment (toVersus) [inline cmd/synthetics/import.go]:
```suggestion
			log.Fatalf("Failed to get synthetics items: %s\n", err)
```

File: cmd/synthetics/import.go [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+		}
+
+		for _, synthetic := range synthetics {
+			if checkNameAndID(synthetic, syns) {
+				disallowUnexpectedProperties(&synthetic)
+				fmt.Printf("CREATE NAME:%s\n", *synthetic.Name)
+				_, err := cli.CreateSyntheticsTest(&synthetic)
+				if err != nil {
+					log.Fatalf("failed to create synthetics items: %s\n", err)
```
Comment (toVersus) [inline cmd/synthetics/import.go]:
```suggestion
					log.Fatalf("Failed to create synthetics items: %s\n", err)
```

File: cmd/synthetics/export.go [hunk 5]
```diff
@@ -0,0 +1,55 @@
+package synthetics
```
Comment (toVersus) [inline cmd/synthetics/export.go]: Add a license header as follows:

```
//
// Licensed under the Apache License, Version 2.0 (the "License");
//
// See Copyright Notice in LICENSE
//
```

File: cmd/synthetics/import.go [hunk 6]
```diff
@@ -0,0 +1,96 @@
+// Copyright © 2017 NAME HERE <EMAIL ADDRESS>
```
Comment (toVersus) [inline cmd/synthetics/import.go]: Change the license header as follows:

```
//
// Licensed under the Apache License, Version 2.0 (the "License");
//
// See Copyright Notice in LICENSE
//
```

File: cmd/synthetics/root.go [hunk 7]
```diff
@@ -0,0 +1,41 @@
+// Copyright © 2017 NAME HERE <EMAIL ADDRESS>
```
Comment (toVersus) [inline cmd/synthetics/root.go]: Change the license header as follows:

```
//
// Licensed under the Apache License, Version 2.0 (the "License");
//
// See Copyright Notice in LICENSE
//
```

File: cmd/synthetics/root.go [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+// syntheticCmd represents the syntheticsCmd
+var syntheticsCmd = &cobra.Command{
+	Use:   "synthetics",
+	Short: "Perform operations related to synthetics of Datadog",
+}
+
+func init() {
+	cmd.RootCmd.AddCommand(syntheticsCmd)
+	// Here you will define your flags and configuration settings.
```
Comment (toVersus) [inline cmd/synthetics/root.go]: Delete unused comments below.

File: cmd/synthetics/show_all.go [hunk 9]
```diff
@@ -0,0 +1,39 @@
+package synthetics
```
Comment (toVersus) [inline cmd/synthetics/show_all.go]: Change the license header as follows:

```
//
// Licensed under the Apache License, Version 2.0 (the "License");
//
// See Copyright Notice in LICENSE
//
```

File: cmd/synthetics/import.go [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+func checkNameAndID(synthetic datadog.SyntheticsTest, syns []datadog.SyntheticsTest) bool {
+	for _, syn := range syns {
+		if *syn.Name == *synthetic.Name || *syn.MonitorId == *synthetic.MonitorId {
+			return false
+		}
+	}
+	return true
+}
+
+// disallowUnexpectedProperties sets nil in properties.
```
Comment (toVersus) [inline cmd/synthetics/import.go]: I think the change in the function name from `disallowUnexpectedProperties` to `maskUnsupportedArguments` or `overwriteUnsupportedArguments` makes more intuitive when describing a role of the function.

Comment (tani-yu) [reply cmd/synthetics/import.go]: Thank you your opinion!
I was just worry with the function name. 
Datadog API return `Additional properties are not allowed....`.
I will change to `maskUnsupportedProperties`.

**Pull request merged**